db/migrate/20210310122746_add_column_merged_by_user_to_patients.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnMergedByUserToPatients < ActiveRecord::Migration[5.2]
+  def change
+    add_column :patients, :merged_by_user_id, :uuid
```

Comment:
Can we add the corresponding model assocation for this? `belongs_to :merged_by_user, class_name: "User"`?

Reply:
We resisted doing this even for `merged_into_patient` because we don't really have a use-case for it yet. YAGNI?

Reply:
Arguably not YAGNI 😅 I'm imagining us opening up a Rails console in a month to debug a recent merge, and not having the convenient methods available to traverse these relationships.

Reply:
I thought we'd be okay not using convenience for prod debugging. But, fair enough, I'll add them in.

Reply:
Fixed in fcefacca2594f4b6211b8cfe62bb255283ac7891.